src/routes/index.tsx

```diff
@@ -42,24 +44,34 @@ const routes: RouteObject[] = [
         element: <AuthGuard component={NameProfile} />,
       },
       {
-        path: 'messages-and-friends',
-        // element: <MessagesAndFriends />,
-        element: <AuthGuard component={MessagesAndFriends} />,
-      },
-      {
-        path: 'messages',
-        element: <Messages />,
-        // element: <AuthGuard component={Messages} />,
-      },
-      {
-        path: 'who-liked-you',
-        element: <YourLikesList />,
-        // element: <AuthGuard component={YourLikesList} />,
-      },
-      {
-        path: 'near-me',
-        element: <NearMe />,
-        // element: <AuthGuard component={NearMe} />,
+        element: <NavBar />,
+        children: [
+          {
+            element: <TabsMessagesFriends />,
+            children: [
+              {
+                path: 'friends',
+                //element: <Friends />,
```

Comment:
Plz comment out lines with AuthGuard, i've made this mistake

Reply:
Done